pubcontrolclient.go

```diff
@@ -97,6 +98,12 @@ func (pcc *PubControlClient) SetAuthJwt(claim map[string]interface{},
 	pcc.lock.Unlock()
 }
 
+func (pcc *PubControlClient) SetAuthBearer(key []byte) {
```

Comment:
This should probably take a string since the value is always casted to one later.